agent/txn_endpoint.go

```diff
@@ -125,10 +125,11 @@ func (s *HTTPServer) convertOps(resp http.ResponseWriter, req *http.Request) (st
 				KV: &structs.TxnKVOp{
```

Comment:
Does there need to be any special treatment of the `case in.Node != nil:` branch or is the default behavior the correct behavior for un-namespaced entities?

Reply:
The default is fine for that branch as nodes only live in the `default` namespace.